scene/resources/world_2d.cpp

```diff
@@ -387,7 +386,7 @@ World2D::World2D() {
 
 	//set space2D to be more friendly with pixels than meters, by adjusting some constants
 	Physics2DServer::get_singleton()->space_set_active(space, true);
-	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 98));
+	Physics2DServer::get_singleton()->area_set_param(space, Physics2DServer::AREA_PARAM_GRAVITY, GLOBAL_DEF("physics/2d/default_gravity", 9.8));
```

Comment:
Note there's also https://github.com/godotengine/godot/blob/69bec86028f87307e549d7a2f49bbb7e2b1f3771/main/tests/test_physics_2d.cpp#L347
But I'm puzzled at why the engine uses 98 in the first place, might be worth checking with @reduz before changing.